src/esp/gfx/RenderTarget.cpp

```diff
@@ -112,8 +147,12 @@ struct RenderTarget::Impl {
 
   void renderEnter() {
     framebuffer_.clearDepth(1.0);
-    framebuffer_.clearColor(0, Mn::Color4{0, 0, 0, 1});
-    framebuffer_.clearColor(1, Mn::Vector4ui{});
+    if (!(flags_ & Flag::NoRgbaBuffer)) {
+      framebuffer_.clearColor(0, Mn::Color4{0, 0, 0, 1});
+    }
+    if (!(flags_ & Flag::NoObjectIdBuffer)) {
```

Comment:
TBH I don't like the double negation everywhere. This would be much easier to read and understand:

```suggestion
    if (flags_ & Flag::ObjectIdBuffer) {
```

Reply:
@mosra : I think about it before I wrote.
The reason is if I use "negative" flags here, yes, it gives me, the developer hard time, and decrease the readability. However it is more friendly in the user's perspective. Because if the user is completely new to the system, she passes an empty flag in, the system by default will give her at least the correct result (It is just a bit slow in terms of performance).

Well, if you use "positive" flags, then the user MUST understand what needs to be set in order to get the correct results. If again, she passes an empty flag in, the system gives here nothing or quits due to the assertions.

I can change it though. Just share what I thought, and I knew you guys would complain it.

Reply:
> Because if the user is completely new to the system, she passes an empty flag in

That was a second part of this thought -- what about providing default `flags` parameter in the constructor, with all set? (That's what I usually do, provide a conservative "always correct" option by default, and let the users override that if they want. But no double negations :))